hawk/app/assets/stylesheets/authentication/_basics.scss

```diff
@@ -8,13 +8,29 @@ html {
 
 body {
   font-family: "Raleway", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
   color: #3a3e40;
   height: 100%;
   cursor: default;
+  background: rgba(5,153,170,1);
+  background: -moz-linear-gradient(left, rgba(5,153,170,1) 0%, rgba(5,85,94,1) 100%);
+  background: -webkit-gradient(left top, right top, color-stop(0%, rgba(5,153,170,1)), color-stop(100%, rgba(5,85,94,1)));
+  background: -webkit-linear-gradient(left, rgba(5,153,170,1) 0%, rgba(5,85,94,1) 100%);
+  background: -o-linear-gradient(left, rgba(5,153,170,1) 0%, rgba(5,85,94,1) 100%);
+  background: -ms-linear-gradient(left, rgba(5,153,170,1) 0%, rgba(5,85,94,1) 100%);
```

Comment:
Commas in function arguments should be followed by a single space<br>Avoid vendor prefixes.